notebooks/official/custom/get_started_vertex_training_xgboost.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Automatically restart kernel after installs so that your environment can access the new packages
</pre><p>I think there's actually an easier way to do this: <code>exit()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>This is from the official template. You may want to make suggestion to Ivan.</p>